Source/CesiumRuntime/Private/Cesium3DTileset.cpp

```diff
@@ -890,6 +890,12 @@ void ACesium3DTileset::UpdateLoadStatus() {
 
   this->LoadProgress = this->_pTileset->computeLoadProgress();
 
+  // If we have tiles to hide next frame, we haven't completely finished loading
+  // yet. We need to tick once more
+  if (!this->_tilesToHideNextFrame.empty()) {
+    this->LoadProgress = glm::min(this->LoadProgress, 99.9999f);
+  }
```

Comment:
Because tile hides are always delayed by one frame, it was possible before for the performance test to end (and suspend further updates) with tiles of multiple LODs visible simultaneously. This change makes it so that loading isn't 100% complete until these tiles have been hidden.